chart/values.yaml

```diff
@@ -402,7 +402,7 @@ components:
       #    fuse: uses fuse-overlayfs
       #    shiftfs: uses shiftfs. Depending on the underlying OS/distribution you
       #             might want to enable the shiftfsModuleLoader.
-      fsShift: shiftfs
+      fsShift: fuse
```

Comment:
Are we sure ? What is the impact.

Reply:
I think this might be the problem with docker/overlayfs.  I didn't mean to include this change in this PR, but it would be good to test if it works. The workspace uses shiftfs to change the uid/gid of files in order to emulate root. fuse-overlayfs is a combination of shiftfs with overlayfs. However, I'm not even sure if this values.yaml file gets used, since there is another one in https://github.com/trilogy-group/DevSpacesNew-installation/tree/master/chart

Reply:
I deleted this commit from the PR, since it was not supposed to be in it.